src/main/java/step2/dto/LottoDTO.java

```diff
@@ -4,13 +4,33 @@
 
 public class LottoDTO {
 
-    private List<Integer> numbers;
+    public static class NormalLottoDTO {
+        private List<Integer> numbers;
 
-    public LottoDTO(List<Integer> numbers) {
-        this.numbers = numbers;
+        public NormalLottoDTO(List<Integer> numbers) {
+            this.numbers = numbers;
+        }
+
+        public List<Integer> getNumbers() {
+            return numbers;
+        }
     }
 
-    public List<Integer> getNumbers() {
-        return numbers;
+    public static class WinningLottoDTO {
+        private List<Integer> numbers;
+        private int bonusNumber;
+
+        public WinningLottoDTO(List<Integer> numbers, int bonusNumber) {
+            this.numbers = numbers;
+            this.bonusNumber = bonusNumber;
+        }
+
+        public List<Integer> getNumbers() {
+            return this.numbers;
+        }
+
+        public int getBonusNumber() {
+            return this.bonusNumber;
+        }
```

Comment:
```suggestion
        public List<Integer> numbers() {
            return this.numbers;
        }

        public int bonusNumber() {
            return this.bonusNumber;
        }
```

dto 내부에서도 get 메서드를 전혀 사용하고 싶지 않고 싶다면,
그냥 담고 있는 멤버명으로 메서드명을 짓거나,
심지어 dto 내부의 멤버를 public final 로 선언하는 것까지도 허용하려 합니다.
더 나아가 dto 나 vo 를 아예 없는 리팩토링도 괜찮다고 생각합니다.

어떤 방식으로 하시던 저는 dto 에서 아래의 2가지만 체크하겠습니다.

1. dto 가 불변 객체인가?
2. dto 내부에 로직이 없는가?